x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -150,7 +164,7 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
         const options: Array<{ label: string }> = [];
 
         fields.forEach((field: Field) => {
-          if (NUMERICAL_FIELD_TYPES.has(field.type) && field.id !== EVENT_RATE_FIELD_ID) {
+          if (shouldAddFieldOption(field)) {
```

Comment:
The comment about filtering for numeric types above here should be updated too, now that the filtering is doing more checks.

Reply:
Updated in c09aa277087cc5f4466d8d2bd1ba19898b890800